clang/lib/Sema/SemaTemplate.cpp

```diff
@@ -2744,31 +2744,155 @@ bool hasDeclaredDeductionGuides(DeclarationName Name, DeclContext *DC) {
   return false;
 }
 
+unsigned getTemplateDepth(NamedDecl *TemplateParam) {
```

Comment:
are we in an anonymous namespace?  Else this should be 'static'.

Reply:
Yeah, we're in an anonymous namespace.